arviz/plots/ppcplot.py

```diff
@@ -1,35 +1,70 @@
 """Posterior predictive plot."""
 import numpy as np
 from .kdeplot import plot_kde, _fast_kde
-from .plot_utils import _scale_fig_size, _create_axes_grid, default_grid
+from .plot_utils import (
+    xarray_var_iter,
+    _scale_fig_size,
+    make_label,
+    default_grid,
+    _create_axes_grid,
+    get_coords,
+)
+from ..utils import _var_names
 
 
-def plot_ppc(
-    data, kind="density", alpha=0.2, mean=True, figsize=None, textsize=None, data_pairs=None
-):
+def plot_ppc(data, var_names=None, coords=None, flatten=None,
+             obs_plot_style="auto", pp_plot_style="auto", pp_flatten="auto",
+             alpha=0.2, mean=True, figsize=None, textsize=None, data_pairs=None):
```

Comment:
`mean` is no longer and argument.

Reply:
See the new commits and discussion below. I reverted and fixed the API changes based on our discussion. The mean parameter is no longer gone.